src/AzureFunctions/Definition.target

```diff
@@ -0,0 +1,21 @@
+<DeploymentTarget>
+	<Id>AZURE_FUNCTIONS</Id>
+	<Name>Microsoft Azure Functions (main procedures)</Name>
+	<Version>1.0</Version>
+	<Description>Deploy to Azure Functions. Only main procedures can be selected.</Description>
+	<PackageTargets>azurefunctions.targets</PackageTargets>
+	<DeployMsbuild>deploy.msbuild</DeployMsbuild>
+	<DeployMsbuildTarget>Deploy</DeployMsbuildTarget>
+	<ResolverFactory>GeneXus.Deployment.AzureFunctionsResolvers.ResolverFactory</ResolverFactory>
+	<Languages>
+		<!-- .Net -->
+		<Language Id="15" ApplicationServer="IIS8"/>
```

Comment:
Do we really support .NET (old Framework)?

Seems that the msbuild implementation only supports .NETCore.

Reply:
Fixed